examples/tutorials/lighting_tutorial.py

```diff
@@ -163,7 +163,11 @@ def main(show_imgs=True, save_imgs=False):
 
     # [example 5]
     light_setup_2 = [
-        LightInfo(position=[8.0, 1.5, 0.0], model=LightPositionModel.GLOBAL)
+        LightInfo(
```

Comment:
There's no obvious conversion between our old hybrid lights and our new point or directional lights, so I'm just tuning new lights from scratch here.